client/components/tinymce/plugins/mentions/suggestion.jsx

```diff
@@ -3,74 +3,67 @@
  */
 import React from 'react';
 
-const Suggestion = React.createClass( {
-	displayName: 'Suggestion',
-
-	propTypes: {
-		avatarUrl: React.PropTypes.string,
-		username: React.PropTypes.string,
-		fullName: React.PropTypes.string,
-		query: React.PropTypes.string
-	},
-
-	getDefaultProps() {
-		return {
-			avatarUrl: '',
-			username: '',
-			fullName: '',
-			query: ''
-		};
-	},
-
-	getRegExpFor( type, textToHighlight ) {
-		const expressions = {};
-
-		expressions.username = '(^' + textToHighlight + ')(\\w*)\\s*';
-		expressions.fullName = '(^.*?)(\\b' + textToHighlight + ')(.*)';
-
-		return new RegExp( expressions[ type ], 'ig' );
-	},
-
-	highlight( content, textToHighlight, type ) {
-		const matcher = this.getRegExpFor( type, textToHighlight ),
-			matches = matcher.exec( content );
-
-		if ( matches ) {
-			const highlights = [];
-			let highlighted = false;
-
-			for ( let i = 1, length = matches.length; i < length; i++ ) {
-				let item = matches[ i ];
-
-				if ( textToHighlight.toLowerCase() === item.toLowerCase() && ! highlighted ) {
-					item = <strong className="mentions__highlight" key={ i }>{ matches[ i ] }</strong>;
-					highlighted = true;
-				}
-
-				highlights.push( item );
-			}
+const getRegExpFor = function( type, textToHighlight ) {
+	const expressions = {};
 
-			return highlights;
-		}
+	expressions.username = '(^' + textToHighlight + ')(\\w*)\\s*';
+	expressions.fullName = '(^.*?)(\\b' + textToHighlight + ')(.*)';
+
+	return new RegExp( expressions[ type ], 'ig' );
+};
+
+const highlight = function( content, textToHighlight, type ) {
+	const matcher = getRegExpFor( type, textToHighlight ),
+		matches = matcher.exec( content );
```

Comment:
we're not using this indenting and comma usage with `let` and `const` and will need a seconds `const` here.